src/PricingManager/Condition/VoucherToken.php

```diff
@@ -59,7 +59,7 @@ public function check(EnvironmentInterface $environment): bool
 
     public function checkVoucherCode(string $code): bool
     {
-        if (in_array(VoucherServiceToken::getByCode($code)->getVoucherSeriesId(), $this->allowListIds)) {
+        if (in_array(VoucherServiceToken::getByCode($code)?->getVoucherSeriesId(), $this->allowListIds)) {
```

Comment:
what about a small refactor to 
```suggestion
        return in_array(VoucherServiceToken::getByCode($code)?->getVoucherSeriesId(), $this->allowListIds);
```
?

Reply:
Done.